pkg/controller/clusterprovision/clusterprovision_controller_test.go

```diff
@@ -198,7 +210,11 @@ func TestClusterProvisionReconcile(t *testing.T) {
 
 			result, err := rcp.Reconcile(reconcileRequest)
 
-			assert.Equal(t, test.expectedReconcileResult, result, "unexpected reconcile result")
+			if test.expectedRequeueAfter {
+				assert.NotEqual(t, time.Duration(0), result.RequeueAfter, "expected requeue after duration in reconcile result")
+			} else {
+				assert.Equal(t, time.Duration(0), result.RequeueAfter, "expected no requeue after duration in reconcile result")
+			}
```

Comment:
It is difficult to set exact expected RequeueAfter duration so just checking whether it is greater than 0 or not. Can it be improved?

Reply:
Store the time before the test structs are created. Store the time after the reconcile is run. Use that to determine a window of values that the RequeueAfter value should lie within.

Reply:
Done